syntax = "proto3";
package cmdb_extend;

option go_package = "go.easyops.local/contracts/protorepo-models/easyops/model/cmdb_extend";





/*
应用依赖
*/
message AppDependency {
message DependServices {
    /*
    模型ID
    */
    string object_id = 1 ;
    /*
    实例ID
    */
    string instance_id = 2 ;
    /*
    名称
    */
    string name = 3 ;
}
        
    
    /*
    应用简称
    */
    string abbreviation = 1 ;
    /*
    应用名称
    */
    string name = 2 ;
    /*
    模型ID
    */
    string object_id = 3 ;
    /*
    实例ID
    */
    string instance_id = 4 ;
    /*
    应用层级
    */
    int32 _hierarchy = 5 ;
    /*
    依赖服务
    */
    repeated DependServices depend_services = 6 ;
}

